src/lib.rs

```diff
@@ -99,10 +99,7 @@ mod errors;
 pub use errors::{DeviceTypeError, EepromStringsError, EepromValueError, FtStatus, TimeoutError};
 
 mod mpsse;
-pub use mpsse::{
-    ClockBits, ClockBitsIn, ClockBitsOut, ClockData, ClockDataIn, ClockDataOut, FtdiMpsse,
-    Ftx232hMpsse, MpsseCmd, MpsseCmdBuilder, MpsseSettings,
-};
```

Comment:
These types should be re-exported from the mpsse crate where applicable, to reduce the number of breaking changes.

Reply:
Fixed: pushed new commit.